src/pet.js

```diff
@@ -0,0 +1,52 @@
+const maxFitness = 10
+const minHunger= 0
+
+class Pet {
+    constructor(name,age,hunger,fitness) {
+        this.name = name;
+        this.age=age;
+        this.hunger=hunger;
+        this.fitness=fitness; 
```

Comment:
Keeping the same formatting (a space after a comma, space either side of an equals sign) would improve readability here
```suggestion
    constructor(name, age, hunger, fitness) {
        this.name = name;
        this.age = age;
        this.hunger = hunger;
        this.fitness = fitness; 
```